InstituteBudget.php

```diff
@@ -1309,9 +1353,9 @@ public function overwriteProceduresOnSOESaved($record) {
 		];
 		$result = \REDCap::saveData($save_params);
 		if (gettype($result) == 'string') {
-			\REDCap::logEvent("TIN Budget Module", "Failed to overwrite Procedures instrument data upon saving Schedule of Event.\r\nREDCap::saveData error: $result");
+			\REDCap::logEvent("Budget Module", "Failed to overwrite Procedures instrument data upon saving Schedule of Event.\r\nREDCap::saveData error: $result");
```

Comment:
consider `private $publicModuleName = "Budget Module"`

Reply:
Updated in various places